orchagent/qosorch.h

```diff
@@ -36,6 +36,15 @@ const string scheduler_algo_STRICT              = "STRICT";
 const string scheduler_weight_field_name        = "weight";
 const string scheduler_priority_field_name      = "priority";
 
+const string scheduler_min_bandwidth_rate_field_name
+                                                = "cir";
+const string scheduler_min_bandwidth_burst_rate_field_name  
+                                                = "cbs";
+const string scheduler_max_bandwidth_rate_field_name        
+                                                = "pir";
+const string scheduler_max_bandwidth_burst_rate_field_name  
+                                                = "pbs";
```

Comment:
Can you put each string to one line? Also, can you add comments to what these acronyms stand for?